src/Kiota.Builder/Writers/CSharp/CSharpConventionService.cs

```diff
@@ -2,49 +2,74 @@
 using System.Collections.Generic;
 using System.Linq;
 using Kiota.Builder.Extensions;
+using static Kiota.Builder.CodeTypeBase;
 
 namespace Kiota.Builder.Writers.CSharp {
     public class CSharpConventionService : CommonLanguageConventionService {
         public override string StreamTypeName => "stream";
         public override string VoidTypeName => "void";
         public override string DocCommentPrefix => "/// ";
-        public override string PathSegmentPropertyName => "PathSegment";
-        public override string CurrentPathPropertyName => "CurrentPath";
-        public override string HttpCorePropertyName => "HttpCore";
-        public HashSet<string> NullableTypes { get; } = new() { "int", "bool", "float", "double", "decimal", "Guid", "DateTimeOffset" };
-        public static string NullableMarker => "?";
+        private const string PathSegmentPropertyName = "PathSegment";
+        private const string CurrentPathPropertyName = "CurrentPath";
+        private const string HttpCorePropertyName = "HttpCore";
+        public HashSet<string> NullableTypes { get; } = new(StringComparer.OrdinalIgnoreCase) { "int", "bool", "float", "double", "decimal", "long", "Guid", "DateTimeOffset" };
```

Comment:
`NullableTypes` could potentially be `private` and `readonly`.

Reply:
thanks! fixed!